engines/python/setup/djl_python/properties_manager/chat_properties.py

```diff
@@ -0,0 +1,75 @@
+from typing import Optional, Union, List, Dict
+
+from pydantic.v1 import BaseModel, Field, validator, root_validator
+
+
+class ChatProperties(BaseModel):
```

Comment:
I think having a strongly typed structure makes sense. I haven't tested this specifically, but does using Pydantic here introduce more overhead (since it's running per request) compared to using just a regular class or dictionary?

I don't think we need to address it in this PR, but we should evaluate other implementations. For this release i think we should mark our chat completions support as experimental.